src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -926,10 +927,19 @@ def setSubscriptionInformation(self, custodialSites = None, nonCustodialSites =
             outputDataset = entry["outputDataset"]
             outputModule = entry["outputModule"]
 
-            primDs = outputDataset.split('/')[1]
-            tier = outputDataset.split('/')[3]
+            dsSplit = outputDataset.split('/')
+
+            primDs = dsSplit[1]
+
+            procDsSplit = dsSplit[2].split('-')
+            skim = ( len(procDsSplit) == 4 )
```

Comment:
> 4 is the number of '-' separated fields in the processed datasets name. The 4th is the filterName and is only used for skims. So if you have 4, the dataset in question is a "skim" by some definiiton. Good enough for the Tier0 at least, as all Tier0 output strictly follows processed dataset naming conventions.

So for non Tier0 workflow,  useSkim is always False, right? 
Also, maybe it is good to have some lexicon check when Tier0 ProcDS is created so it checks the convention (if it doesn't have it yet)